charts/vector-experimental/templates/daemonset.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if (eq .Values.role "Agent") -}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "vector.fullname" . }}
```

Comment:
Do you think we could make the namespace configurable? Pretty sure users will request it. If we do we should also include the namespace in the other k8s resources the chart creates.

Reply:
That's always something that interested me - I always felt like it didn't need to specified in templates because you can just pass `--namespace` to the helm install/upgrade commands. Maybe I'm missing something?

Reply:
Yeah actually i remembered we had it in the agent chart, but we just set it to `{{ .Release.Namespace }}` so it's not really useful, and we rely on `--namespace` as well 👌

Reply:
Cool - it always seemed like an unneeded line with the CLI option (other than say RoleBindings that need the namespace outside of the metadata)